app/models/foreman_virt_who_configure/config.rb

```diff
@@ -97,6 +97,9 @@ class Config < ActiveRecord::Base
     validates :hypervisor_password, :presence => true, :if => Proc.new { |c| c.hypervisor_type != 'libvirt' && c.hypervisor_type != 'kubevirt' }
     validates :hypervisor_username, :presence => true, :if => Proc.new { |c| c.hypervisor_type != 'kubevirt' }
     validates :hypervisor_server, :presence => true, :if => Proc.new { |c| c.hypervisor_type != 'kubevirt' }
+    #validates :kubeconfig_path, :absence => true, :if => Proc.new { |c| c.hypervisor_type != 'kubevirt' }
+    validates :kubeconfig_path, :absence => {:message => "Can not be used with non Kubevirt hypervisor type." }, :if => Proc.new { |c| c.hypervisor_type != 'kubevirt' }
```

Comment:
what's the difference between this line and the next line?

Reply:
101 is the line we want so we return an error message to the user.